code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/OpenLDAP/OpenLDAPIdentityProvider.java

```diff
@@ -77,13 +78,19 @@
 
     private Set<Identity> getIdentities(LdapName dn) {
         Set<Identity> identities = new HashSet<>();
-        Attributes userAttributes;
+        Attributes userAttributes, opAttributes = null;
         LdapContext context = getServiceContext();
         try {
             userAttributes = context.getAttributes(dn);
             if (!hasPermission(userAttributes)){
                 throw new ClientVisibleException(ResponseCodes.UNAUTHORIZED);
             }
+            try {
+                String[] operationalAttrList = {"1.1", "+", "*"};
```

Comment:
what are these attributes? they look pretty random to me :) is there a documentation? or can it be referenced from ldap third party library?